src/ontology/cl-edit.owl

```diff
@@ -32234,6 +32236,26 @@ SubClassOf(obo:CL_4042016 obo:CL_0000313)
 SubClassOf(obo:CL_4042016 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005385))
 SubClassOf(obo:CL_4042016 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0011148))
 
+# Class: obo:CL_4047001 (small intestine BEST4+ enterocyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24223998") Annotation(oboInOwl:hasDbXref "PMID:34497389") obo:IAO_0000115 obo:CL_4047001 "An enterocyte of the human small intestine expressing bestrophin-4 (BEST4) calcium-activated ion channels.")
+AnnotationAssertion(terms:contributor obo:CL_4047001 <https://orcid.org/0009-0005-7919-4905>)
+AnnotationAssertion(terms:date obo:CL_4047001 "2024-07-25T13:05:50Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4047001 "small intestine BEST4+ enterocyte")
+SubClassOf(obo:CL_4047001 obo:CL_4030026)
+SubClassOf(obo:CL_4047001 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002108))
+SubClassOf(obo:CL_4047001 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_Q8NFU0))
+
+# Class: obo:CL_4047002 (BEST4+ colonocyte)
```

Comment:
consider to add the related synonym
has related synonyms
BEST4 mature colonocyte https://elifesciences.org/reviewed-preprints/88879#

Reply:
`BEST4+ mature colonocyte` should be a related synonym to the term `BEST4+ colonocyte `and not `BEST4+ enterocyte`